.github/workflows/ci-macarm.yml

```diff
@@ -0,0 +1,154 @@
+# to have more fine-grained control over when it runs.
+
+# This section controls when the workflow runs.
+on:
+  # Run upon push, but only for tagged versions, or if the branch name
+  # contains the substrings "macarm", "-alpha", "-beta", or is called
+  # "release".
+  push:
+    tags:
+      - v**
+    branches:
+      - '*macarm*'
+      - '*alpha*'
+      - '*beta*'
+      - 'release'
+  # Run for pull requests whose branch name includes "macarm" (this allows
+  # us to test specific PRs that we think need ARM verification).
+  pull_request:
+    branches:
+      - '*macarm*'
+  # Run monthly on the 27th at 8:00AM to make sure we haven't broken
+  # anything among the many PRs that didn't test on ARM.
+  schedule:
+    - cron: "0 8 27 * *"
+      if: github.repository == 'AcademySoftwareFoundation/OpenColorIO'
+      
+permissions: read-all
+
+jobs:
+  # ---------------------------------------------------------------------------
+  # macOS
+  # ---------------------------------------------------------------------------
+
+  macos-arm:
+    name: 'macOS 13 arm
+      <AppleClang 
+       arch=${{ matrix.arch-type }},
+       config=${{ matrix.build-type }}, 
+       shared=${{ matrix.build-shared }}, 
+       simd=${{ matrix.use-simd }}, 
+       cxx=${{ matrix.cxx-standard }}, 
+       python=${{ matrix.python-version }}, 
+       docs=${{ matrix.build-docs }}, 
+       oiio=${{ matrix.use-oiio}}>'
+    # Needs special runners, only run on the main repo,
+    if: github.repository == 'AcademySoftwareFoundation/OpenColorIO'
+    runs-on: macos-13-xlarge
+    strategy:
+      matrix:
+        build: [1]
+        include:
+          - build: 1
+            arch-type: "arm64"
+            build-type: Release
+            build-shared: 'ON'
+            build-docs: 'OFF'
+            build-openfx: 'OFF'
+            use-simd: 'ON'
+            use-oiio: 'OFF'
+            cxx-standard: 11
+            python-version: '3.11'
+    steps:
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Install docs env
+        run: share/ci/scripts/macos/install_docs_env.sh
+        if: matrix.build-docs == 'ON'
+      - name: Install tests env
+        run: share/ci/scripts/macos/install_tests_env.sh
+      - name: Create build directories
+        run: |
+          mkdir _install
+          mkdir _build
+      - name: Configure
+        run: |
+          cmake ../. \
+                -DCMAKE_INSTALL_PREFIX=../_install \
+                -DCMAKE_BUILD_TYPE=${{ matrix.build-type }} \
+                -DCMAKE_CXX_STANDARD=${{ matrix.cxx-standard }} \
+                -DBUILD_SHARED_LIBS=${{ matrix.build-shared }} \
+                -DOCIO_BUILD_DOCS=${{ matrix.build-docs }} \
+                -DOCIO_BUILD_OPENFX=${{ matrix.build-openfx }} \
+                -DOCIO_BUILD_GPU_TESTS=OFF \
```

Comment:
If I understand correctly, these runners have GPU acceleration enabled, should we set this value to ON to run the metal unit tests?

Reply:
Great point, Remi.

Reply:
@chandyn , please set GPU_TESTS to ON when you get a chance and we will go ahead and merge.